draft-ietf-wimse-s2s-protocol.md

```diff
@@ -536,6 +538,15 @@ so it cannot be easily reused in another context, which is often a security impr
 be designed to include extensibility to sign other fields, but this would make it closer to
 trying to reinvent Message Signatures.
 
+## Error Conditions
+
+Errors may occur during the processing of the message signature or WPT. If the signature verification fails for any reason,
+such as an invalid signature, an expired validity time window, or a malformed data structure, an error is returned. Typically,
+this will be in response to an API call, so an HTTP status code such as 400 (Bad Request) is appropriate. This response could
+include more details as per {{RFC7807}} and {{RFC9457}}, such as an indicator that the wrong key material or algorithm was used.
+If the WPT or message signature is used for another purpose, such as authentication to start a session, a different response may
+be more appropriate, such as an HTTP status code of 401 (Unauthorized) or 403 (Forbidden).
```

Comment:
This sentence is problematic IMO for two reasons:

- We don't define the concept of a "session", rather we assume a classic REST model where each request is independent of the others. And of course each WPT or message signature provides workload authentication.
- This still does not address my question in #15 regarding the `Authorization` header.

Reply:
I'm happy to remove that part of the sentence.  I agree that it is problematic.  I don't think we can put anything in the authorization header as we do not use and leave it reserved for another token.